adsmp/models.py

```diff
@@ -77,7 +77,7 @@ class Records(Base):
     metrics_checksum = Column(String(10), default=None)
     datalinks_checksum = Column(String(10), default=None)
     
-    status = Column(Enum('solr-failed', 'metrics-failed', 'links-failed', 'success', name='status'))
+    status = Column(Enum('solr-failed', 'metrics-failed', 'links-failed', 'retrying', 'success', name='status'))
```

Comment:
This is a database schema change, I expect it to require an alembic script that would take care of adding/removing the new enum value.

Reply:
We aren't defining an enum type at the postgres or alembic level, we are just using a string(255).  https://github.com/adsabs/ADSMasterPipeline/blob/master/alembic/versions/4475ef3e98af_created_db_structure.py#L42